package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "big.js": "^5.2.2",
```

Comment:
- You don't need `big.js` in this milestone. please remove it to keep your dependencies clean
- Add `redux` and `react-redux` to the list of packages. You will need those in the next milestone.